contrib/clojure-package/README.md

```diff
@@ -64,6 +62,14 @@ sudo apt-get update
 sudo apt install libopencv-imgcodecs3.4
 ```
 
+You may also need
```

Comment:
By may, do you mean it is optional to install them. If so is it because of the Clojure API?